Bucketeer/Sources/Public/BKTClient.swift

```diff
@@ -55,7 +55,7 @@ public class BKTClient {
         }
     }
 
-    private func execute(_ handler: @escaping () throws -> Void) {
+    func execute(_ handler: @escaping () throws -> Void) {
```

Comment:
Changed: the fun visibility from `private` to `internal` because 
- we need access to it from the e2e test 
https://github.com/bucketeer-io/ios-client-sdk/pull/51/files#diff-2c2e38d4c144a27cc9e8e9ea9dc39345dca55c1d293246e59f7fe70fb18f9882R90
- we could only access to the evaluation data from the `EvaluationStorage`